### DISCUSSION TITLE
[NHẬN TASK]

### DISCUSSION THREAD

**Discussion by nguyenxuanquang6789** [Runway-Club/Runway-CV-Builder#21] [Task Delivery]
Cho mình xin task: Tạo giao diện trang landingpages và thông tin các nhà tuyển dụng

**Comment by nguyentantrung2000** [comment]
Chào bạn hiện tại trang Landing Page còn nhiều sửa chửa về mặt thiết kế , nên tạm thời bạn chọn task khác nhé
